default/python/AI/FreeOrionAI.py

```diff
@@ -341,6 +341,9 @@ def generateOrders():  # pylint: disable=invalid-name
                    ResourcesAI.generate_resources_orders,
                    ]
 
+    from freeorion_tools import assertion_fails
+    print assertion_fails(False)
+    print assertion_fails(True)
```

Comment:
Are these leftover from some initial testing?

Reply:
@Morlic I presume I am right that these were just accidentally left over, and they are making spurious message window spam with each turn, so I've pushed a commit now removing these lines.

Reply:
yes, sorry and thanks for fixing.